blog/zh/blog/2022/10/28/apisix-ingress-with-horizon-ai.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "从 Traefik 到 APISIX，汽车智能计算平台公司「地平线」在 Ingress Controller 的探索和实践"
+authors:
+  - name: "张昕"
+    title: "Author"
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://github.com/hf400159.png"
+keywords: 
+- 汽车智能
+- AI 芯片
+- 地平线
+- Apache APISIX
+- Ingress
+description: 地平线作为一家致力于汽车自动驾驶芯片的公司，在业务迭代中为何从 Traefik 切换到了 APISIX Ingress 去支撑业务。
+tags: [Case Studies]
+---
+
+> 地平线作为一家致力于汽车自动驾驶芯片的公司，在业务迭代中为何从 Traefik 切换到了 APISIX Ingress 去支撑业务？
+
+<!--truncate-->
+
+> 作者张昕，地平线云原生开发工程师。
```

Comment:
I don't think you need to mention it twice, it is already written in the `authors` part.

Reply:
Need to mention the author's title to prove the content is empirical (someone has asked for it, as you know)